admin/inc/functions_themes.php

```diff
@@ -420,7 +420,7 @@ function cache_stylesheet($tid, $filename, $stylesheet)
 		"theme" => $theme_directory
 	);
 	$stylesheet = parse_theme_variables($stylesheet, $theme_vars);
-	$stylesheet = preg_replace_callback("#url\((\"|'|)(.*)\\1\)#", create_function('$matches', 'return fix_css_urls($matches[2]);'), $stylesheet);
+	$stylesheet = preg_replace_callback("#url\((\"|'|)(.*)\1\)#", create_function('$matches', 'return fix_css_urls($matches[2]);'), $stylesheet);
```

Comment:
[`create_function()`](https://secure.php.net/create_function) cannot be used as it will be deprecated with PHP 7.2. See https://github.com/mybb/mybb/issues/2743

Reply:
Yep, got to sort out the conflicts.